util/chplenv/chpl_llvm.py

```diff
@@ -324,7 +324,8 @@ def get_sysroot_resource_dir_args():
     args = [ ]
     target_platform = chpl_platform.get('target')
     llvm_val = get()
-    if target_platform == "darwin" and llvm_val == "bundled":
+    if (target_platform == "darwin" and
+        (llvm_val == "bundled" or llvm_val == "system")):
```

Comment:
I don't think it's worth changing, but just checking for my sake -- is this the same as `llvm_val != 'none'`?

Reply:
Not quite, because we also can have 'unset' as the CHPL_LLVM value.